site/content/docs/latest/background/cryptography.md

```diff
@@ -43,7 +74,7 @@ The public key (in the form of a self-signed certificate if it was generated by
 > Note that it is possible to use your own X509 certificate with the command bellow:
 
 ```console
```

Comment:
I imagine this is `shell` too
```suggestion
```shell
```

Reply:
I reviewed the code examples in the page, including the command output where it provides extra information to users.